src/v2/guide/filters.md

```diff
@@ -36,6 +36,32 @@ Vue.filter('capitalize', function (value) {
 })
 ```
 
+Below is an example of our `capitalize` filter being used:
+
+{% raw %}
+<div id="example_1" class="demo">
```

Comment:
Sorry for a late question (just found that when do translation), but is it better to use hyphenated id (`example-1`) as other doc pages used?
Thanks :-)

Reply:
@Jinjiang Yeah, good catch!